Rubberduck.RegexAssistant/i18n/AssistantResources.de.resx

```diff
@@ -109,126 +89,126 @@
   </data>
   <data name="AtomDescription_Literal_UnicodePoint" xml:space="preserve">
-    <value>Erkennt den Unicode-Codepoint {0}</value>
+    <value>Gleicht gegen den Unicode Codepoint {0} ab</value>
   </data>
   <data name="AtomDescription_Newline" xml:space="preserve">
-    <value>Erkennt das ASCII-Zeichen LF, auch als "Neue Zeile" bekannt</value>
+    <value>Gleicht gegen das ASCII-Zeichen  LF, bekannt als Zeilenumbruch" ab</value>
   </data>
   <data name="AtomDescription_NonDigit" xml:space="preserve">
-    <value>Erkennt Zeichen, die keine Ziffern sind. Äquivalent zu "[^\d]"</value>
+    <value>Akzeptiert alles außer Ziffern. Äquivalent zu "[^0-9]"</value>
   </data>
   <data name="AtomDescription_NonWhitespace" xml:space="preserve">
-    <value>Erkennt Zeichen, die nicht Leerzeichen sind. Äquivalent zu "[^\s]"</value>
+    <value>Akzeptiert alles außer Abstandszeichen. Äquivalent zu "[^\s]"</value>
   </data>
   <data name="AtomDescription_NonWordBoundary" xml:space="preserve">
-    <value>Stellt sicher, dass die aktuelle Position eine "Nicht-Wortgrenze" ist</value>
+    <value>Stellt sicher, dass die aktuelle Position keine Wortgrenze ist</value>
   </data>
   <data name="AtomDescription_NonWordCharacter" xml:space="preserve">
-    <value>Erkennt "nicht-wort Zeichen". Äquivalent zu "[^\w]"</value>
+    <value>Akzeptiert alles außer "Wortzeichen". Äquivalent zu "[^\w]"</value>
   </data>
   <data name="AtomDescription_VTab" xml:space="preserve">
-    <value>Erkennt das vertikale "Tab"-Zeichen</value>
+    <value>Gleicht gegen das vertikale "Tab"-Zeichen ab</value>
   </data>
   <data name="AtomDescription_Whitespace" xml:space="preserve">
-    <value>Erkennt ein beliebiges "Leerzeichen". Äquivalent zu"[ \t\r\n\v\f]"</value>
+    <value>Akzeptiert ein beliebiges Abstandszeichen. Äquivalent zu "[\t\r\n\v\f]"</value>
   </data>
   <data name="AtomDescription_WordBoundary" xml:space="preserve">
     <value>Stellt sicher, dass die aktuelle Position eine Wortgrenze ist</value>
   </data>
   <data name="AtomDescription_WordCharacter" xml:space="preserve">
-    <value>Erkennt ein beliebiges "Wort-Zeichen". Äquivalent zu "[a-zA-Z_0-9]"</value>
+    <value>Akzeptiert ein beliebiges "Wortzeichen". Äquivalent zu "[a-zA-Z_0-9]"</value>
   </data>
   <data name="ExpressionDescription_AlternativesExpression" xml:space="preserve">
-    <value>Erkennt eine von {0} Alternativen</value>
+    <value>Akzeptiert eine von {0} Alternativen</value>
   </data>
   <data name="ExpressionDescription_ConcatenatedExpression" xml:space="preserve">
-    <value>Erkennt die exakte Sequenz</value>
+    <value>Akzeptiert exakt die Sequenz</value>
   </data>
   <data name="ExpressionDescription_ErrorExpression" xml:space="preserve">
-    <value>Konnte '{0}' nicht als Literal erkennen. Bitte überprüfen Sie die Eingabe</value>
+    <value>Konnte '{0}' nicht als Literal erkennen. Bitte Überprüfen Sie die Eingabe.</value>
```

Comment:
That should be lower case.
Furthermoer, nothing here except this sentance seems to contain a period at the end.

Reply:
I think I'll revert that whole file. It wasn't correctly included in the solution, which caused me to retranslate it completely after switching toolstack